src/main/java/com/taosdata/jdbc/rs/RestfulConnection.java

```diff
@@ -36,6 +38,42 @@ public RestfulConnection(String host, String port, Properties props, String data
         this.metadata = new RestfulDatabaseMetaData(url, props.getProperty(TSDBDriver.PROPERTY_KEY_USER), this);
     }
 
+    /**
+     * A convenient constructor for cloud user
+     * @param url TDengine Cloud URL
+     * @param token TDengine Cloud Token
+     * @throws Exception
+     */
+    public RestfulConnection(String url, String token) throws Exception {
+        super(new Properties());
+        clientInfoProps.setProperty(TSDBDriver.PROPERTY_KEY_TIMESTAMP_FORMAT, String.valueOf(TimestampFormat.TIMESTAMP));
+
+        int poolSize = Integer.parseInt(HttpClientPoolUtil.DEFAULT_MAX_PER_ROUTE);
+        HttpClientPoolUtil.init(poolSize, true);
+        if (url == null || token == null) {
+            throw new Exception("url and token can't be null");
+        }
+        if (!url.startsWith("http://") && !url.startsWith("https://")) {
+            System.out.println("No protocol specified in url, use \"http://\" by default");
+        }
+
+        this.url = url;
+        this.useSsl = url.startsWith("https://");
+        String tmpSchema = this.useSsl ? "https://" : "http://";
+        String hostPort = url.replaceFirst(tmpSchema, "");
+        String[] splits = hostPort.split(":");
```

Comment:
you can use 'com.taosdata.jdbc.AbstractDriver#parseURL' to parse url.

Reply:
It's too heavy for this simple processing